### PR TITLE
docs: Add buildkit install guide

### PR DESCRIPTION
**Pull request by felipehuici**

### CODE REVIEW

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Building Dockerfile Images with Buildkit
+description: |
+  This guide shows how to install buildkit to allow you to build Dockerfile images with Unikraft.
+---
+Unikraft uses BuildKit, along with Dockerfiles, to construct the root filesystem used by applications.
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:6]: Add empty newline above this.

Add a link to BuildKit and Dockerfile, i.e `Unikraft uses [BuildKit](https://docs.docker.com/build/buildkit/), along with [Dockerfiles](https://docs.docker.com/reference/dockerfile/) ...`

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:8]: Only use one blank line.

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+You can install BuildKit by either running it natively via systemd or in a
+container. Please refer to [BuildKit's installation
+instructions](https://github.com/moby/buildkit) for more information.
+
+To use BuildKit with Docker on your system, you first need to install Docker
+following the [official
+instructions](https://docs.docker.com/engine/install/). This can be either
+[Docker Engine](https://docs.docker.com/engine/) or [Docker
+Desktop](https://docs.docker.com/desktop/).
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:17]: Use one and only one sentence per line. When a period appears (`.`), you start a newline. We have the documentation guidelines buried somewhere, we will dig them out.

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+After installing Docker, you can run BuildKit in a container via:
+
+```bash
+docker run -d --name buildkitd --privileged moby/buildkit:latest
+```
+
+Now instruct `kraft` to use the BuildKit instance from the `buildkitd` container
+by either setting the environmental variable:
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:26]: Only use one line.

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks, @felipehuici , looks good. See my comments.

File: content/guides/building-dockerfile-images-with-buildkit.mdx (hunk 1 above)
Comment (felipehuici) [reply content/guides/building-dockerfile-images-with-buildkit.mdx:6]: Thanks, will fix.

File: content/guides/building-dockerfile-images-with-buildkit.mdx (hunk 2 above)
Comment (felipehuici) [reply content/guides/building-dockerfile-images-with-buildkit.mdx:8]: Thanks, will fix.

File: content/guides/building-dockerfile-images-with-buildkit.mdx (hunk 3 above)
Comment (felipehuici) [reply content/guides/building-dockerfile-images-with-buildkit.mdx:17]: Thanks, will fix.

File: content/guides/building-dockerfile-images-with-buildkit.mdx (hunk 4 above)
Comment (felipehuici) [reply content/guides/building-dockerfile-images-with-buildkit.mdx:26]: Thanks, will fix.

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+title: Building Dockerfile Images with Buildkit
+
+description: |
+  This guide shows how to install buildkit to allow you to build Dockerfile images with Unikraft.
+---
+Unikraft uses [BuildKit](https://docs.docker.com/build/buildkit/), along with [Dockerfiles](https://docs.docker.com/reference/dockerfile/), to construct the root filesystem used by applications.
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:7]: Add an empty new line above this.

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+You can install BuildKit by either running it natively via systemd or in a container. Please refer to [BuildKit's installation instructions](https://github.com/moby/buildkit) for more information.
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:9]: Add newline after period (`.`).

File: content/guides/building-dockerfile-images-with-buildkit.mdx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+To use BuildKit with Docker on your system, you first need to install Docker following the [official instructions](https://docs.docker.com/engine/install/). This can be either [Docker Engine](https://docs.docker.com/engine/) or [Docker Desktop](https://docs.docker.com/desktop/).
```
Comment (razvand) [inline content/guides/building-dockerfile-images-with-buildkit.mdx:11]: Add newline after period (`.`).

**Review by razvand — CHANGES_REQUESTED** [review]
Just some small comments and we should be OK.

Do a squash of all commits to only have one commit.

**Review by razvand — APPROVED** [review]
Thanks, @felipehuici 

Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**